frontend/src/metabase/App.jsx

```diff
@@ -1,34 +1,41 @@
 /* @flow weak */
 
-import React, { Component } from "react";
```

Comment:
FYI there's no point in changing minor formatting like this, it will just get changed back when we enable `prettier`